config/routes.rb

```diff
@@ -67,6 +67,12 @@
       get 'ping', to: 'pings#show'
       post 'login', to: 'logins#login_user'
 
+      if FeatureToggle.enabled?('PHONE_NUMBER_MASKING')
+        scope :exotel_sessions do
+          get 'create', to: 'exotel_sessions#create'
```

Comment:
Seeing `get` with a `create` action is weird. I think we should leave a comment here say it's because of exotel calls using get.